tests/src/test/javascript/msltests.html

```diff
@@ -70,6 +70,7 @@
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/WebCryptoUsage.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/CipherKey.js"></script>
 <script type="text/javascript" src="../../../../core/src/main/javascript/crypto/PublicKey.js"></script>
+<script type="text/javascript" src="../../../../core/src/main/javascript/crypto/KeyFormat.js"></script>
```

Comment:
It had worked as previously defined, and now I realize the reason why.  All scripts that referenced KeyFormat were not immediately evaluated - not until after KeyFormat.js was included.

I moved it above 'WebCryptoAdapter.js' which is the first script that uses it.